.bazelrc

```diff
@@ -1,4 +1,6 @@
-build --extra_toolchains=@gcc_toolchain//:toolchain --incompatible_enable_cc_toolchain_resolution
+build --incompatible_enable_cc_toolchain_resolution
+build --extra_toolchains=@gcc_toolchain//:toolchain
+build --extra_toolchains=@gcc_toolchain_armv7//:toolchain
```

Comment:
is this redundant with the toolchain being "registered"?

Reply:
No. If this is not given, the toolchain resolution silently picks the default host toolchain.